code/modules/mob/living/simple_mob/subtypes/animal/space/alien.dm

```diff
@@ -83,170 +81,627 @@
+		M.apply_damage(2 * damage, BRUTE, BP_TORSO)
+		M.apply_damage(0.5 * damage, BRUTE, BP_L_LEG)
+		M.apply_damage(0.5 * damage, BRUTE, BP_R_LEG)
+		M.apply_damage(0.5 * damage, BRUTE, BP_L_ARM)
+		M.apply_damage(0.5 * damage, BRUTE, BP_R_ARM)
+		blood_splatter(src, M, 1)
+
+/mob/living/simple_mob/animal/space/alien/monarch/apply_melee_effects(atom/A)
+	if(isliving(A))
+		var/mob/living/L = A
+		if(L.mob_size <= MOB_MEDIUM)
+			visible_message(SPAN_DANGER("\The [src] sends \the [L] flying with their heavy claws!"))
+			playsound(src, "sound/mobs/biomorphs/breaker_slam.ogg", 50, 1)
+			var/throw_dir = get_dir(src, L)
+			var/throw_dist = L.incapacitated(INCAPACITATION_DISABLED) ? 4 : 1
+			L.throw_at_old(get_edge_target_turf(L, throw_dir), throw_dist, 1, src)
+		else
+			to_chat(L, SPAN_WARNING( "\The [src] punches you with incredible force, but you remain in place."))
+
+/mob/living/simple_mob/animal/space/alien/breaker/death()
+	..()
+	visible_message("[src] emits a high pitched roar as its massive body stills, acidic blood pouring from its remains.")
+	playsound(src, 'sound/mobs/biomorphs/breaker_death_hiss.ogg', 100, 1)
 
-	mod_min = 100
-	mod_max = 150
+/mob/living/simple_mob/animal/space/alien/monarch/death()
+	..()
+	visible_message("[src] lets out a horrifying screech that echoes throughout your mind, it seems like it's finally over... Or is it?")
+	playsound(src, 'sound/mobs/biomorphs/monarch_death_hiss.ogg', 100, 1)
 
 /mob/living/simple_mob/animal/space/alien/death()
 	..()
 	visible_message("[src] lets out a waning guttural screech, green blood bubbling from its maw...")
-	playsound(src, 'sound/voice/hiss6.ogg', 100, 1)
+	playsound(src, 'sound/mobs/biomorphs/xenomorph_death_hiss.ogg', 100, 1)
+
+/mob/living/simple_mob/animal/space/alien/special/burrower
+	name = "xenomorph burrower"
+	icon = 'icons/mob/biomorphs/burrower.dmi'
+	desc = "A utter abomination which appears to be some sort of mesh between a spider and a Xenomorph."
+	icon_state = "burrow_walk"
+	icon_living = "burrow_walk"
+	icon_dead = "burrow_dead"
+	icon_rest = "burrow_sleep"
+	maxHealth = 300
+	health = 300
+	legacy_melee_damage_lower = 35
+	legacy_melee_damage_upper = 40
+	faction = "xeno"
+	movement_cooldown = 0
+	icon_scale_x = 0.7
+	icon_scale_y = 0.7
+	base_pixel_x = -16
+	base_pixel_y = -5
+	attack_sound =  'sound/weapons/bite.ogg'
+	ai_holder_type = /datum/ai_holder/polaris/simple_mob/melee/tunneler
+
+	// Tunneling is a special attack, similar to the hunter's Leap.
+	special_attack_min_range = 2
+	special_attack_max_range = 6
+	special_attack_cooldown = 10 SECONDS
+
+	var/tunnel_warning = 0.5 SECONDS	// How long the dig telegraphing is.
+	var/tunnel_tile_speed = 2			// How long to wait between each tile. Higher numbers result in an easier to dodge tunnel attack.
+
+/mob/living/simple_mob/animal/giant_spider/tunneler/frequent
+	special_attack_cooldown = 5 SECONDS
+
+/mob/living/simple_mob/animal/giant_spider/tunneler/fast
```

Comment:
Remember to fix the type if you're copy-pasting code. This is still typed to the spiders.